# CRISPR-base-editor
Expanding the flexibility of base editing for high-throughput genetic screens in bacteria

Abstract: Genome-wide screens have become powerful tools for elucidating genotype-to-phenotype relationships in bacteria. Of the varying techniques to achieve knockout and knockdown, CRISPR base editors are emerging as promising options. However, the limited number of available, efficient target sites hampers their use for high-throughput screening. Here, we make multiple advances to enable flexible base editing as part of high-throughput genetic screening in bacteria. We first co-opt the Streptococcus canis Cas9 that exhibits more flexible PAM recognition than the traditional Streptococcus pyogenes Cas9. We then expand beyond introducing premature stop codons by mutating start codons. Next, we derive guide design rules by applying machine learning to an essentiality screen conducted in Escherichia coli. Finally, we rescue poorly edited sites by combining base editing with Cas9-induced cleavage of unedited cells, thereby enriching for intended edits. The efficiency of this dual system was validated through a conditional essentiality screen based on growth in minimal media. Overall, expanding the scope of genome-wide knockout screens with base editors could further facilitate the investigation of new gene functions and interactions in bacteria.Â 

This GitHub contains codes and files related to this study. Detailed steps for analysis can be found in the method section in the paper and the README files in each folder.
